dashing/distribution.yaml

```diff
@@ -94,6 +94,7 @@ repositories:
       url: https://github.com/ros2-gbp/ament_cmake-release.git
       version: 0.7.4-1
     source:
+      test_abi: true
```

Comment:
What is the ABI test actually doing for this repo which only contains CMake stuff?

Reply:
the short answer is: nothing. The only point of having it enable for cmake code is to see how the auto-abi tool behaves when it does not find any library or header. It should not fail. So my propose here was to test the non-failure feature of running the tool on non c/c++ code.